### DISCUSSION TITLE
`select blocks to calculate` seemed not work

### DISCUSSION THREAD

**Discussion by yoghurtdewdew** [YU000jp/logseq-plugin-some-menu-extender#40] [General]
`select blocks to calculate` seemed not work? Or more information for using this command?
<img width="231" alt="image" src="https://github.com/YU000jp/logseq-plugin-some-menu-extender/assets/63794438/80562d85-c0c4-4fbf-828e-d2d31f3b839c">

**Comment by YU000jp** [comment]
![SUM](https://github.com/YU000jp/logseq-plugin-some-menu-extender/assets/111847207/ee09c4a9-933d-4fea-9f5b-a655669ef67d)
This is a program that performs addition between blocks containing only numbers. First, select the first block and press the Esc key, then use the Ctrl key to select each subsequent block. Please move the block with the total value to any desired location by dragging it with the mouse.

**Reply by yoghurtdewdew** [reply]
Got it. But it would be hard for me to leave blocks with only numbers without description. Would be better for calculate blocks containing not only numbers.
Thanks for your great work!
